Aquality.Selenium.Core/tests/Aquality.Selenium.Core.Tests/Applications/WindowsApp/Elements/WindowElement.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Aquality.Selenium.Core.Applications;
+using Aquality.Selenium.Core.Elements;
+using Aquality.Selenium.Core.Elements.Interfaces;
+using Aquality.Selenium.Core.Localization;
+using Aquality.Selenium.Core.Utilities;
+using Aquality.Selenium.Core.Waitings;
+using Microsoft.Extensions.DependencyInjection;
+using OpenQA.Selenium;
+
+namespace Aquality.Selenium.Core.Tests.Applications.WindowsApp.Elements
```

Comment:
There are some rules that you use to define should the test be in the core or in the particular implementation?

Reply:
I believe that I cannot test WebDriver features properly without having a WebDriver.
Mocking could take ages, and we would work with specific implementations anyways, with all the bugs and features that they bring to us.